### ISSUE TITLE
Missing actions (DAX)

### ISSUE THREAD

**Issue by alexcasalboni** [aletheia/iam-policy-generator#21]
Working on a new example based on DAX, I found these missing actions in the generated code:

```json
[
  "dax:DefineAttributeList",
  "dax:DefineAttributeListId",
  "dax:DefineKeySchema",
  "dax:Endpoints"
]
```

Can you please verify why they aren't included?

**Comment by aletheia** [comment]
Just checked our `policy.json` file that is missing them as well. Going deeper into the issue, it looks like even AWS missed them in the IAM Policy Generator console (which relies on AWS IAM internal API https://console.aws.amazon.com/iam/api/policies).

Unfortunately, with no external reference to automatically generate them, you have to enter them manually.

**Comment by alexcasalboni** [comment]
I see, thanks. Ok, I'd propose to keep working on #19 so we get more visibility into how that JSON file changes over time :)

**Comment by aletheia** [comment]
Is going to be solved by #19 .

**Comment by aletheia** [comment]
Solved by #19
